tests/e2e_scenarios.py

```diff
@@ -38,7 +38,7 @@ def run(args):
             check = infra.checker.Checker()
             check_commit = infra.checker.Checker(mc)
             with primary.user_client() as uc:
-                check_commit(uc.rpc("mkSign"), result=True)
+                uc.rpc("mkSign")
```

Comment:
Why are we no longer waiting for commit here?

Reply:
I'm not sure why we even trigger a signature here, and as you know, we cannot usefully wait for this particular commit. What we could do is sample commit, issue a sig, and wait for it to go up. Unless someone remembers why we issue a sig here, I'm just going to remove it.